bosc.php

```diff
@@ -12,7 +12,9 @@
 
 
         <?php if ( have_posts() ) : while ( have_posts() ) : the_post(); ?>
-        <?php the_content(); ?>
+        <h5 style="color:black;">
```

Comment:
The css changes for <h5> tag is done globally. That's why I thought it would be better if we can   update only this local block of code which will fix the issue.

Reply:
Hmmm - how about adding a class for this?